articles/chaos-studio/chaos-studio-limitations.md

```diff
@@ -15,6 +15,7 @@ During the public preview of Azure Chaos Studio, there are a few limitations and
 
 ## Limitations 
 
+* The target resources must be in [one of the regions supported by the Azure Chaos Studio Preview](https://azure.microsoft.com/en-us/global-infrastructure/services/?products=chaos-studio) 
```

Comment:
please remove the locale-specific portion of the url (en-us)

Reply:
Good catch, thx!